runtime/service/npud/core/Backend.h

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONE_SERVICE_NPUD_CORE_BACKEND_H__
+#define __ONE_SERVICE_NPUD_CORE_BACKEND_H__
+
+#include "ir/Layout.h"
+#include "ir/DataType.h"
+
+#include <string>
+#include <vector>
+#include <memory>
+#include <map>
+
+namespace npud
+{
+namespace core
+{
+
+/* Backend supports up to 16 buffers. */
+#define NPU_TENSOR_CARDINALITY_MAX (16)
```

Comment:
It seems like backend interface is trix-specific. 
can be abstract more ?

Reply:
Please ignore this comment. I will investigate more.

Reply:
I updated this value with more general name.